src/globals/objects/_grid.scss

```diff
@@ -1,53 +1,16 @@
-@mixin govuk-grid {
-  @include govuk-clearfix;
-  margin-right: - ($govuk-gutter-half);
-  margin-left: - ($govuk-gutter-half);
-}
-
-@mixin govuk-grid-item {
-  box-sizing: border-box;
-  padding: 0 $govuk-gutter-half;
-
-  @include mq($from: tablet) {
-    float: left;
-  }
-}
-
-@include govuk-exports("grid") {
-
-  .govuk-grid {
-    @include govuk-grid;
-  }
-
-  .govuk-grid__item {
-    @include govuk-grid-item;
-  }
-
-  .govuk-grid__item--full {
-    width: 100%;
-  }
-
-  .govuk-grid__item--one-half {
-    @include mq($from: tablet) {
-      width: percentage(1 / 2);
-    }
-  }
-
-  .govuk-grid__item--one-third {
-    @include mq($from: tablet) {
-      width: percentage(1 / 3);
-    }
-  }
-
-  .govuk-grid__item--two-thirds {
-    @include mq($from: tablet) {
-      width: percentage(2 / 3);
-    }
-  }
-
-  .govuk-grid__item--one-quarter {
-    @include mq($from: tablet) {
-      width: percentage(1 / 4);
-    }
-  }
+@include govuk-exports("grid-system") {
+  //most common usage
+  @include govuk-layout-row;
+  @include govuk-layout-col(one-quarter);
+  @include govuk-layout-col(one-third);
+  @include govuk-layout-col(one-half);
+  @include govuk-layout-col(two-thirds);
+  @include govuk-layout-col(three-quarters);
+  @include govuk-layout-col(full);
+
+  // custom 100,66,75%
+  @include govuk-layout-col(three-quarters, $at: desktop, $class: "on-desktop");
```

Comment:
Should we have custom examples as part of the core library? It seems good to demonstrate but we shouldn't include them unless we know they'll be used?

Reply:
no for demo only. still WIP :)